app/src/main/java/com/bogdan/codeforceswatcher/adapter/ContestAdapter.kt

```diff
@@ -38,6 +41,12 @@ class ContestAdapter(private var items: List<Contest>, private val ctx: Context)
         val calendarEventLink = "$CALENDAR_LINK?action=TEMPLATE&text=$encodeName&dates=$timeStart/$timeEnd&details=$CODEFORCES_LINK"
         val intent = Intent(Intent.ACTION_VIEW).setData(Uri.parse(calendarEventLink))
         ctx.startActivity(intent)
+
+        val bundle = Bundle()
+        bundle.putString(FirebaseAnalytics.Param.ITEM_ID, "id")
```

Comment:
I believe, this event type doesn't fit well for our app. Consider using custom event type, like this one:
```
val params = Bundle()
params.putString("contest_name", contest.name)
firebaseAnalytics.logEvent("add_contest_to_google_calendar", params)
```